internal/config/app.go

```diff
@@ -105,11 +106,12 @@ func getDefaultConfig() *AppConfig {
 			Password: "password",
 		},
 		Password: Password{
-			Length:                 8,
-			ResetExpirationMinutes: 15 * time.Minute,
+			Length:              8,
+			ResetLinkExpiration: 15 * time.Minute,
```

Comment:
It might be a standard situation for Go language to use some numbers in the code without description, but for other languages, it is not. In my opinion, it is better to use a variable with a proper name to describe why we use this magic number here.

Reply:
I didn't have the aim to change it, so I made only slight modifications.
And this solution looks clear, we have only 1 place where these magic numbers are used.